### ISSUE TITLE
many console messages "Tuple.append/2 is deprecated. Use insert_at instead"

### ISSUE THREAD

**Issue by beep-beep-beep-boop** [Qqwy/elixir-type_check#209]
Hi,
when I run my project that uses TypeCheck, the console gets spammed with thousands of lines of this warning:

```
    warning: Tuple.append/2 is deprecated. Use insert_at instead
    │                                                                              
 62 │               altered_param = Tuple.append(altered_param, altered_element)   
    │                                                                              
    └─ deps/type_check/lib/type_check/builtin/fixed_tuple.ex:62                    
```

It's spamming so many of these errors that I think the I/O may actually be slowing down the program, but I can't figure out how to disable them to test it.
